holoviews/operation/datashader.py

```diff
@@ -362,7 +362,7 @@ def _process(self, element, key=None):
 
         if self.p.precompute:
             self._precomputed[element._plot_id] = x, y, data, glyph
-        (x_range, y_range), (xs, ys), (width, height), (xtype, ytype) = self._get_sampling(element, x, y)
+        (x_range, y_range), (xs, ys), (width, height), (xtype, ytype) = self._get_sampling(element, x, y, apply_pixel_ratio=False)
```

Comment:
Without this, it'll error below on line 374 because width/height mismatch len(xs) and len(ys)

Reply:
This defeats the point of the pixel ratio entirely as far as I can tell.

Reply:
Yeah, the correct fix here is to multiply the width/height before creating the coordinates, not to simply disable pixel ratio entirely.